### ISSUE TITLE
Support for custom HTTP post bodies in a handler file or in TICK script

### ISSUE THREAD

**Issue by nikist97** [influxdata/kapacitor#2036]
Currently, as described in here: https://docs.influxdata.com/kapacitor/v1.5/event_handlers/post/#options ,
the HTTP post body cannot be customized in a handler file or when using .post() in TICK script.

I think it is sensible to be able to customize this at least in a handler file, too, and not just limit the customization to be available in the kapacitor.conf file only.

**Comment by quaff** [comment]
I thought message would be http post body, wasted me lots of time, :(

**Comment by nikist97** [comment]
Nope, it's not, it just adds a message field in the default JSON HTTP post body.

**Comment by caengcjd** [comment]
Oh,what the pity!

**Comment by caengcjd** [comment]
custom http post arguments  in the tickscript is the strong demand but you guys ignore it.You team should need a product manager.

**Comment by quaff** [comment]
@nathanielc Please implement this.

**Comment by quaff** [comment]
ping

**Comment by AkaiNoCat** [comment]
Feel like it should be a good feature to supports custom all http post body
Really looking forward to~

**Comment by pravindamodaran** [comment]
Any idea if this is implemented?
